File: parse_excel.py
import pandas as pd
import json

try:
    file_path = r"c:\Users\neton\OneDrive\Projetos IA\Antigravit\dAIGAPEstim\Machine\Template - BRDC Complexity Definition ABAP (PT).xls"
    xls = pd.ExcelFile(file_path)
    
    print("Sheets available:", xls.sheet_names)
    
    for sheet_name in xls.sheet_names:
        df = pd.read_excel(file_path, sheet_name=sheet_name)
        print(f"\n--- Sheet: {sheet_name} ---")
        # Print first 50 rows to get the structure
        print(df.head(50).to_string())
        
except Exception as e:
    print("Error:", e)

File: parse_excel.js
const XLSX = require('xlsx');

try {
    const workbook = XLSX.readFile('Machine/Template - BRDC Complexity Definition ABAP (PT).xls');
    console.log("Sheets:", workbook.SheetNames);
    
    // Ler a primeira aba
    const sheetName = workbook.SheetNames[0];
    const sheet = workbook.Sheets[sheetName];
    const data = XLSX.utils.sheet_to_json(sheet, { header: 1 });
    
    for (let i = 0; i < Math.min(data.length, 100); i++) {
        console.log(`Row ${i}:`, data[i]);
    }
} catch (e) {
    console.error("Error:", e);
}
